extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/TenantContextFactory.java

```diff
@@ -547,8 +530,9 @@ public Uni<OidcProviderClientImpl> apply(OidcConfigurationMetadata metadata, Thr
                                         "The application supports RP-Initiated Logout but the OpenID Provider does not advertise the end_session_endpoint"));
                             }
                         }
-                        if (userInfoInjectionPointDetected && metadata.getUserInfoUri() != null) {
-                            enableUserInfo(oidcConfig);
+                        if (userInfoInjectionPointDetected && metadata.getUserInfoUri() != null
```

Comment:
The UserInfo acquisition does not always depend on the presence of the injected `@UserInfo`.
For example, `@Authenticated` alone should work if the access token must be verified indirectly via UserInfo acquisition

Reply:
logic behind this change didn't change, Sergey, if you want, I'll put this method back as it was, this PR, what you see, it changes nothing regarding this line....

Reply:
I'll just put back what was there so that we don't need to discuss it, but really, this code does 100 % same thing. I simply thought this reads better.....

Reply:
and it's back, it is identical code that was there before now, no changes, so I am closing this comment thread.